index.js

```diff
@@ -23,6 +24,18 @@ var new_client = function(api_key, config) {
   client.api_key = api_key;
   client.queue = [];
   client.offline = false;
+  client.proxy_host = config.proxy_host;
+  client.proxy_port = config.proxy_port;
+  client.proxy_auth = config.proxy_auth;
+
+  // Initialize global tunnel if proxy options are set
+  if (client.proxy_host && client.proxy_port) {
```

Comment:
Identifier 'proxy_host' is not in camel case.<br>Identifier 'proxy_port' is not in camel case.